lib/workers/repository/init/merge.spec.ts

```diff
@@ -209,9 +211,11 @@ describe('workers/repository/init/merge', () => {
         configFileName: '.github/renovate.json',
         configFileParsed: {},
       });
+      GlobalConfig.reset();
```

Comment:
do it in after or before each, otherwise it's bubbling to other tests on failure